modules/shared_vpc_access/variables.tf

```
@@ -35,3 +35,9 @@ variable "active_apis" {
   type        = list(string)
   default     = []
 }
+
+variable "gke_shared_vpc_security_admin_role_enabled" {
```

We could probably make this slightly more generic as future services might also need it.

```suggestion
variable "grant_services_security_admin_role" {
```

---

Done.